### ISSUE TITLE
Default attribute values

### ISSUE THREAD

**Issue by jeanphilippegoldman** [nlplab/brat#1310]
If I define a different attribute for each entity, with a defalut value for each attribute, then Brat tries to add every default values for an entitite. E.g

[entities]
noun   
verb   
[attributes]   
genre         Arg:noun, Value:masc|fem, \<DEFAULT\>:masc   
transitivity  Arg:verb, Value:transitive|intransitive, \<DEFAULT\>:transitive

then, annotating a word as noun for example, gives me an error like this image.
See how the attribute 'transivity' is added to noun categories

Did I miss stg in conf files ? or there's stg wrong ? Any idea ?
Thanks in advance

![image](https://user-images.githubusercontent.com/14802091/55638661-0629f700-57c8-11e9-8411-45d4bfe10d7b.png)

**Comment by aroao** [comment]
How did you manage to have a default attribute? I've tried using the same structure as yours but it doesn't work:

P	Arg:<a>, Value:x|y|z, <DEFAULT>:x

Is there anything wrong? Thanks in advance

**Comment by oya163** [comment]
@aroao You might want to update your brat version

@jeanphilippegoldman The default value is being propagated to other entity types. Have you found any solutions yet??

**Comment by jeffmax** [comment]
I don't think this problem is isolated to default values. If, for example, you select an Entity Type that supports a Boolean attribute and then turn the attribute on, but then switch to another Entity Type that does not support that Boolean attribute, it will cause a similar error. I believe the fix for this would be to place the following code:

```javascript
if ($.inArray(attrNo, spanTypes[typeRadio.val()].attributes || []) == -1){
    return;
}
```
right above this [line](https://github.com/nlplab/brat/blob/master/client/src/annotator_ui.js#L2389)
This fix requires some testing though because if it is not 100% correct it could prevent saving of attributes the user has clicked.

**Comment by Hui-zju** [comment]
The default value is being propagated to other entity types.  Why this bug still exist now?
